portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -56,12 +56,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-          String UserComment = request.getParameter("comment");
-          String user = request.getParameter("user");
+    String UserComment = request.getParameter("comment");
```

Comment:
Is there any chance that the parameters are not provided? You might want to add some checks for them in order to not break your program.